resources/config-defaults.edn

```diff
@@ -292,10 +292,15 @@
  ;; Show "Assign external id" action button in UI for handlers
  :enable-assign-external-id-ui false
 
- ;; maximum size of attachment files in bytes (10000000 = 10MB)
+ ;; Maximum size of attachment files in bytes (10000000 = 10MB).
  ;; nil = no limit
  :attachment-max-size 10000000
 
+ ;; Should draft applications be deleted after a set period of time has passed?
+ :enable-expired-draft-application-processing false
+ ;; Expiration threshold in days as number
+ :draft-application-expiration-threshold 90
```

Comment:
name(s) pending, suggestions are welcomed

Reply:
More future proof would be e.g. `:application-expiration {:draft "P3M"}` or so. (https://en.wikipedia.org/wiki/ISO_8601#Durations is used in `:email-retry-period` and could be nice or just `-days`)

Reply:
`:enable-expired-draft-application-processing` sounds like something can happen besides deletion. Also perhaps we can have the treshold/expiration as `nil` = never so there is no need for another flag?

Reply:
super good point, i didn't realize we had this sort of duration already expressed. :+1: one flag should be enough for this case